src/ontology/cl-edit.owl

```diff
@@ -3255,7 +3255,7 @@ AnnotationAssertion(rdfs:label obo:IAO_0000115 "definition")
 
 AnnotationAssertion(rdfs:label obo:IAO_0000424 "expand expression to")
 
-# Annotation Property: obo:IAO_0000700 (preferred_root)
+# Annotation Property: obo:IAO_0000700 (has ontology root term)
```

Comment:
@anitacaron this keeps bouncing from preferred_root to has ontology root term, and vice versa. Same with synonyms. How can we prevent this from happening? See https://github.com/obophenotype/cell-ontology/pull/2401/files fro the opposite change

Reply:
Need to remove the declarations from the edit file :)